### PR TITLE
AB#3335 -- Added contact us to apply for your child screen

### PR DESCRIPTION
**Pull request by AdamP-HRSDC-RHDCC**
### Description
Added contact us to apply for your child screen (if under 16 with a child)

### Related Azure Boards Work Items
[AB#3335](https://dev.azure.com/DTS-STN/1fc40a8f-28cf-47bc-b6e4-1c234bd06177/_workitems/edit/3335)

### Screenshots (if applicable)
image

### Checklist
- [x] I have tested the changes locally
- [x] I have updated the documentation if necessary
- [ ] I have added/updated tests that prove my fix is effective or that my feature works
- [x] I have checked that my code follows the project's coding style by running `npm run format:check`
- [x] I have checked that my code contains no linting errors by running `npm run lint`
- [x] I have checked that my code contains no type errors by running `npm run typecheck`
- [x] I have checked that all unit tests pass by running `npm run test:unit -- run`
- [x] I have checked that all e2e tests pass by running `npm run test:e2e`

### Test Instructions
Select applying for myself/child and enter dob before 16 years of age to confirm new page

### Additional Notes
n/a

### CODE REVIEW

File: frontend/app/routes.json [hunk 1]
```diff
@@ -248,6 +248,14 @@
                       "en": "/:lang/apply/:id/living-independently",
                       "fr": "/:lang/appliquer/:id/living-independently"
                     }
+                  },
+                  {
+                    "id": "$lang+/_public+/apply+/$id+/contact-apply-child",
+                    "file": "routes/$lang+/_public+/apply+/$id+/contact-apply-child.tsx",
+                    "paths": {
+                      "en": "/:lang/apply/:id/contact-apply-child",
+                      "fr": "/:lang/appliquer/:id/contact-apply-child"
```
Comment (nicholas-ly) [inline frontend/app/routes.json:257]: I'm not bilingual, but using Google translate:

```suggestion
                      "fr": "/:lang/appliquer/:id/contact-demande-enfant"
```

File: frontend/public/locales/en/apply.json [hunk 2]
```diff
@@ -94,6 +94,15 @@
     "exit-link": "https://www.canada.ca/en/services/benefits/dental/dental-care-plan.html",
     "status-checker-link": "https://www.canada.ca/en/services/benefits/dental/dental-care-plan/apply.html"
   },
+  "contact-apply-child": {
+    "page-title": "Contact us to apply for your child",
+    "contact-service-canada": "You must speak to a Service Canada representative if you are under 16 and applying for your child. Please contact Service Canada by calling 1-833-537-4342 or visit a <serviceCanadaOfficeLink>Service Canada Office</serviceCanadaOfficeLink>.",
```
[the remark this answers is not in the document]
Comment (AdamP-HRSDC-RHDCC) [reply frontend/public/locales/en/apply.json]: Added nowrap span

**Review by Fbasham — APPROVED** [review]
Approving on behalf of Nicholas.  Changes have been made and I'm happy with them.

**Comment by gregory-j-baker** [comment]
image

Rebasing this branch on top of main would be preferrable to merging main into this branch.

**Review by gregory-j-baker — APPROVED** [review]

**Pull request merged**